src/test/regress/expected/isolation_get_all_active_transactions.out

```diff
@@ -35,8 +35,8 @@ step s2-begin-insert:
 
 step s3-as-admin:
  -- Admin should be able to see all transactions
- SELECT count(*) FROM get_all_active_transactions();
- SELECT count(*) FROM get_global_active_transactions();
+ SELECT count(*) FROM get_all_active_transactions() WHERE datid != 0;
```

Comment:
Why we needed these changes, do we get datid == 0 rows back?

Reply:
I'm updating to transaction_number